locations/Bahraich/inputs/Optimisation/optimisations.yaml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Perhaps `optimisation_inputs.yaml` and `optimisations.yaml` could be combined into a single file. What do you think? Would it cause any issues under the new structure?

Reply:
I don't think it would? This would mean that the `optimisation_inputs.yaml` file contains the list of optimisations as well as the optimisation parameters:
```
iteration_length: 4 # [years]
number_of_iterations: 3 # number of iterations to carry out
pv_size:
  min: 1 # [kWp]
  max: 1 # [kWp]
  step: 1 # [kWp]
storage_size:
  min: 1 # [kWh]
  max: 1 # [kWh]
  step: 1 # [kWh]
optimisations:
  - optimisation_criteria:
      - lcue: minimise # Name of the column and whether to maximise/minimise.
    threshold_criteria:
      - blackouts: 0.1 # Max/min value permitted (see guidance)
```

Reply:
I'll combine them and we can see what we think :smile: